FoodAPI/AppAPI/urls.py

```diff
@@ -2,17 +2,25 @@
 from . import views
 from rest_framework import routers
 
-
+"""
```

Comment:
Why are there comment blocks here?

Reply:
They were used to block out older code but I will remove it now.